app/controllers/users/registrations_controller.rb

```diff
@@ -0,0 +1,34 @@
+module Users
+  class RegistrationsController < ApplicationController
+    before_action :ensure_tribunal_case
```

Comment:
We've had a convo with JB, & decided to remove the "create an account" link from the signin form. So, this code can stay as it is.